### ISSUE TITLE
ask your questions here

### ISSUE THREAD

**Issue by iperov** [iperov/DeepFaceLive#11]

**Comment by ykk648** [comment]
What's meaning of 'two pass' in FaceSwapper which did't existed on DeepFaceLab.

**Comment by fortminors** [comment]
@iperov Is there a way to run the program from source instead of using the prebuilt windows binary?

**Comment by iperov** [comment]
yes, but I don't support any issues with that.

You can run _internal\vscode.bat to view and edit the code.

**Comment by iperov** [comment]
@fortminors and keep in mind I will not accept PRs 🤣

**Comment by fortminors** [comment]
@iperov ok makes sense. Another question: would I need to train a new model for each face that I want to swap (in DeepFaceLab)?

**Comment by iperov** [comment]
I am writing FAQ about that

**Comment by gjd2017** [comment]
how to deal with landmarks shake?

**Comment by Th3DEAD** [comment]
Would it be possible for the stream to continue without stopping as soon as there is no face detected ?

**Comment by iperov** [comment]
@Th3DEAD and reveal original face?

**Comment by Th3DEAD** [comment]
Good point 👍

**Comment by jasonlbx13** [comment]
how can I run it on Ubuntu?

**Comment by iperov** [comment]
it is possible, but I don't support linux.

**Comment by liyy201912** [comment]
@iperov Thanks for your nice work. Where can I find a sample onnx model (say Tom_Cruise.onnx) to test the app?

**Comment by iperov** [comment]
nowhere.

![firefox_2021-08-03_12-34-06](https://user-images.githubusercontent.com/8076202/127984551-4be36456-46d2-4a93-a774-17dd1c920902.png)

**Comment by Th3DEAD** [comment]
I'm training a model using your small guide here, but there is no "Learning rate dropout" with AMP SRC-SRC or AMP.
It's all right to skip it ? Thanks.
